pkg/gofr/datasource/mongo/mongo.go

```diff
@@ -89,14 +87,19 @@ func (c *Client) Connect() {
 	m, err := mongo.Connect(context.Background(), options.Client().ApplyURI(uri))
 	if err != nil {
 		c.logger.Errorf("error connecting to mongoDB, err:%v", err)
+		return
+	}
 
+	if err = m.Ping(context.Background(), nil); err != nil {
+		c.logger.Errorf("error connecting mongoDB %s, err:%v", c.config.URI, err)
 		return
 	}
 
 	mongoBuckets := []float64{.05, .075, .1, .125, .15, .2, .3, .5, .75, 1, 2, 3, 4, 5, 7.5, 10}
 	c.metrics.NewHistogram("app_mongo_stats", "Response time of MONGO queries in milliseconds.", mongoBuckets...)
 
 	c.Database = m.Database(c.config.Database)
+	c.logger.Logf("connecting to mongoDB at %v to database %v", c.config.URI, c.config.Database)
```

Comment:
this should be in the beginning as we tried to mention that we have started the connection - and here after it actually connected you can add comment for connected.